### PR TITLE
Allow Symfony 7 & simplify testing workflow

### PR DESCRIPTION
**Pull request by jmsche**
Hi, 

This PR allows Symfony 7, and simplifies the testing workflow by adding Symfony & PHP versions in a matrix :smile:

### CODE REVIEW

File: .github/workflows/test.yaml [hunk 1]
```diff
@@ -22,121 +22,45 @@ jobs:
...
-                  composer require --no-update --dev symfony/framework-bundle=$SYMFONY_VERSION symfony/yaml=$SYMFONY_VERSION
-                  composer update --prefer-dist --no-interaction --no-ansi --no-progress
-                  php vendor/bin/simple-phpunit
+        strategy:
+            matrix:
+                php-version:
+                    - '8.1'
+                    - '8.2'
+                symfony-version:
+                    - '5.4.*'
```
Comment (tgalopin) [inline .github/workflows/test.yaml:38]: I think we can keep a test suite for 6.0, 6.1 and 6.2, it's basically free. No need to drop support as long as it work without need to update.

**Review by tgalopin — COMMENTED** [review]
Thanks! A quick comment but I like the PR :) !

File: .github/workflows/test.yaml (hunk 1 above)
Comment (jmsche) [reply .github/workflows/test.yaml:38]: Done, also added PHP 8.3 :)

**Review by maxhelias — APPROVED** [review]

**Comment by maxhelias** [comment]
Thanks @jmsche !

**Comment by maxhelias** [comment]
I've created a new release : https://github.com/thephpleague/flysystem-bundle/releases/tag/3.3.0

**Comment by jmsche** [comment]
Thanks for the quick tagging :smile:

**Pull request merged**